src/validator.ts

```diff
@@ -83,5 +79,4 @@ export class FormValidator {
             errorMsg.remove();
         }
     }
-
 }
```

Comment:
dodaj pusta linie na koncu pliku. 
Wszedzie gdzie widzisz taki symbol to znaczy ze brakuje entera na samym koncu pliku. Z jakiegos powodu Git to weryfikuje i potem ma problem z tym :D 
<img width="281" alt="image" src="https://github.com/apawlik96/formGenerator/assets/112213953/e311932c-8607-457d-bb3e-bfbff05e2c35">